markdown/developers/adding_modules.md

```diff
@@ -69,6 +71,8 @@ We have implemented a number of commands in the `nf-core/tools` package to make
 
 6. Create a module using the [nf-core DSL2 module template](https://github.com/nf-core/tools/blob/master/nf_core/module-template/modules/main.nf):
 
+    > ⚠️ [2021-11-26] please use `dev` version of tools
```

Comment:
This should be maybe also stated in step 1 above.

Reply:
good point - done!